lib/rubocop/cop/rspec/repeated_description.rb

```diff
@@ -47,14 +58,18 @@ def repeated_descriptions(node)
           grouped_examples =
             RuboCop::RSpec::ExampleGroup.new(node)
               .examples
-              .group_by(&:doc_string)
+              .group_by { |example| example_signature(example) }
 
           grouped_examples
-            .select { |description, group| description && group.size > 1 }
+            .select { |signatures, group| !signatures.empty? && group.size > 1 }
```

Comment:
Maybe `signatures.any?`?
In this case `compact` below becomes redundant.

Reply:
Fixed